google-cloud-spanner/src/test/java/com/google/cloud/spanner/ITSessionPoolIntegrationTest.java

```diff
@@ -97,28 +98,28 @@ public ScheduledExecutorService get() {
 
   @Test
   public void sessionCreation() {
-    try (Session session = pool.getReadSession()) {
-      assertThat(session).isNotNull();
+    try (PooledSessionFuture session = pool.getReadSession()) {
+      assertThat(session.get()).isNotNull();
     }
 
-    try (Session session = pool.getReadSession()) {
-      assertThat(session).isNotNull();
-      Session session2 = pool.getReadSession();
-      assertThat(session2).isNotNull();
+    try (PooledSessionFuture session = pool.getReadSession()) {
+      assertThat(session.get()).isNotNull();
+      PooledSessionFuture session2 = pool.getReadSession();
```

Comment:
nit: move `session2` declaration into the `try` with resources block

Reply:
Done.